components/builder-datastore/hooks/run

```diff
@@ -1,17 +1,14 @@
-#!/bin/sh
-#
-
-export PGDATA="{{pkg.svc_data_path}}"
+#!/bin/bash
 
 exec 2>&1
 
 chmod 0700 "{{pkg.svc_data_path}}"
+export PGDATA="{{pkg.svc_data_path}}"
 
```

Comment:
do we need this? Isn't this taken care of by the data_directory setting in postgresql.conf?

Reply:
I believe that we do in that we need the /hab/svc/bujilder-datastore/data directory specified in commands however I will double check my self. I know that most commands specify the data directory.

Reply:
Removed.  I thought this was in the install hook when we were discussing it.